server/src/main/java/com/project/yozmcafe/service/LikedCafeService.java

```diff
@@ -25,13 +25,14 @@ public LikedCafeService(final CafeRepository cafeRepository,
         this.memberService = memberService;
     }
 
-    public List<LikedCafeThumbnailResponse> findLikedCafeThumbnailsByMemberId(final String memberId, final Pageable pageable) {
+    public List<CafeThumbnailResponse> findLikedCafeThumbnailsByMemberId(final String memberId, final Pageable pageable) {
         final Member member = memberService.findMemberByIdOrElseThrow(memberId);
 
         final List<LikedCafe> likedCafes = getLikedCafes(pageable, member);
 
         return likedCafes.stream()
-                .map(LikedCafeThumbnailResponse::from)
+                .map(LikedCafe::getCafe)
```

Comment:
애초에 `Member`에서 List<Cafe>를 받을 수 없을까 고민해보다가 옛날 코드들을 보게 됐는데요!

1. `Member` 내부 `getLikedCafesSection` 메서드에 `reverseLikedCafes`라고 돼있는데 그냥 `likedCafes`라고 하는게 나을 것 같네요...!
2. `LikedCafe`를 서비스에서 사용하기보다는 `Member` 내부로 숨기는 것이 나을 것 같아요. `Member`에서 애초에 `List<Cafe>`를 리턴하는 것이 좋을 것 같다는 생각입니다.
3. 아래의 `getLikedCafes` 내부의 첫 번째 줄은 `pageable.getOffset()` 메서드를 활용하면 깔끔해질 것 같네요!

위의 사항을 다 적용하면,
```suggestion
public List<CafeThumbnailResponse> findLikedCafeThumbnailsByMemberId(final String memberId, final Pageable pageable) {
        final Member member = memberService.findMemberByIdOrElseThrow(memberId);

        final List<Cafe> likedCafes = member.getLikedCafes(pageable.getOffset(), pageable.getPageSize());

        return likedCafes.stream()
                .map(CafeThumbnailResponse::from)
                .toList();
    }
```
이러한 형태가 될 것 같고 `LikedCafeService#getLikedCafes` 메서드는 필요가 없어보여요!

Reply:
이거 고쳤는데 LikedCafeServiceTest쪽 테스트가 실패하더라고요!
왜인지 보니 원래 transactional을 붙여서 테스트 코드에서 변경감지를 이용했었던 걸 transactional을 뺀 후 수정하지 않아서 같아요. 
변경감지를 이용해서 작성한 테스트 코드가 더 있을 수도...?